src/Console/Application.php

```diff
@@ -298,14 +311,14 @@ public function buildContainer()
 
         $c['violation_checker'] = function ($c) {
             return new ComposedViolationChecker(
-                [
+                array(
```

Comment:
This change needs to be done even if this feature were rejected. Otherwise the application would not run on PHP 5.3.